sixMans/sixMans.py

```diff
@@ -681,234 +763,143 @@ async def setDefaultTeamSelection(self, ctx, team_selection_method):
-        queue = None
-        for queue in self.queues:
-            if channel in queue.channels:
-                queue.channels.remove(channel)
-                break
-        if queue.channels:
-            return
-        
-        clone = await channel.clone()
-        helper_role = await self._helper_role(channel.guild)
-        helper_ping = " {}".format(helper_role.mention) if helper_role else ""
-        await clone.send(":grey_exclamation:{} This channel has been created because the last textChannel for the **{}** queue has been deleted.".format(helper_ping, queue.name))
-        queue.channels.append(clone)
-        await self._save_queues(channel.guild, self.queues)
+#endregion
 
+#region helper methods
 
-    async def has_perms(self, ctx):
+    async def has_perms(self, ctx: Context):
         helper_role = await self._helper_role(ctx.guild)
         if ctx.author.guild_permissions.administrator:
             return True
         elif helper_role and helper_role in ctx.author.roles:
             return True
 
-    async def _add_to_queue(self, player, six_mans_queue):
+    async def _add_to_queue(self, player: discord.Member, six_mans_queue: SixMansQueue):
         six_mans_queue._put(player)
-        player_list = self._format_player_list(six_mans_queue)
-
-        embed = discord.Embed(color=discord.Colour.green())
-        embed.set_author(name="{0} added to the {1} queue. ({2}/{3})".format(player.display_name, six_mans_queue.name,
-            six_mans_queue.queue.qsize(), team_size), icon_url="{}".format(player.avatar_url))
-        embed.add_field(name="Players in Queue", value=player_list, inline=False)
-
-        for channel in six_mans_queue.channels:
-            await channel.send(embed=embed)
+        embed = self.embed_player_added(player, six_mans_queue)
+        await six_mans_queue.send_message(embed=embed)
 
-    async def _remove_from_queue(self, player, six_mans_queue):
+    async def _remove_from_queue(self, player: discord.Member, six_mans_queue: SixMansQueue):
         six_mans_queue._remove(player)
-        player_list = self._format_player_list(six_mans_queue)
+        embed = self.embed_player_removed(player, six_mans_queue)
+        await six_mans_queue.send_message(embed=embed)
 
-        embed = discord.Embed(color=discord.Colour.red())
-        embed.set_author(name="{0} removed from the {1} queue. ({2}/{3})".format(player.display_name, six_mans_queue.name,
-            six_mans_queue.queue.qsize(), team_size), icon_url="{}".format(player.avatar_url))
-        embed.add_field(name="Players in Queue", value=player_list, inline=False)
-
-        for channel in six_mans_queue.channels:
-            await channel.send(embed=embed)
-
-    async def _auto_remove_from_queue(self, player, six_mans_queue):
+    async def _auto_remove_from_queue(self, player: discord.Member, six_mans_queue: SixMansQueue):
         try:
             await self._remove_from_queue(player, six_mans_queue)
```

Comment:
If the player can't be found or the bot can't send a dm to them we don't want an error thrown here because it causes the looped task to end if an error is thrown pretty sure